test/e2e/theia_clickhouse_test.go

```diff
@@ -270,8 +272,8 @@ func testTheiaGetClickHouseInsertRate(t *testing.T, data *TestData, connect *sql
 		assert.Equal(4, len(tableInfoArray), "tableInfoArray: %s", tableInfoArray)
 		actualInsertRate, error := strconv.Atoi(tableInfoArray[1])
 		assert.NoError(error)
-		tableNum := len(tableColumnNumberMap)
-		percent := (actualInsertRate/tableNum - recordPerCommit/insertInterval) * 100 / (recordPerCommit / insertInterval)
+		percent := (actualInsertRate/flowsRelatedTableNum - recordPerCommit/insertInterval) * 100 / (recordPerCommit / insertInterval)
+		percent = int(math.Abs(float64(percent)))
 		assert.LessOrEqualf(percent, threshold, "stdout: %s, expectedInsertRate: %s", stdout, recordPerCommit/insertInterval)
```

Comment:
I think these 3 lines of code could be simplified using `assert.InDeltaf`.
Reference: we are comparing the throughput difference should be less than 15% in https://github.com/antrea-io/theia/blob/main/test/e2e/flowvisibility_test.go#L760

Reply:
Thanks for the suggestion.